plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/parser/SQLParserContext.java

```diff
@@ -74,7 +103,24 @@ public IDocument getDocument() {
     }
 
     public SQLDialect getDialect() {
```

Comment:
By the name, it looks like a getter, but it instantiates an uninitialized script dialect. Getters should not do such things.